samples/features/security/ledger/azure-sql-database/setup/CreateDatabaseSchema.sql

```diff
@@ -0,0 +1,194 @@
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[Employees](
+	[EmployeeID] [int] IDENTITY(1,1) NOT NULL,
+	[SSN] [char](11) NOT NULL,
+	[FirstName] [nvarchar](50) NOT NULL,
+	[LastName] [nvarchar](50) NOT NULL,
+	[Salary] [money] NOT NULL
+	)
+WITH 
+(
+  SYSTEM_VERSIONING = ON,
+  LEDGER = ON
+); 
+GO
+
+
+CREATE TABLE [dbo].[AuditEvents](
+	[Timestamp] [Datetime2] NOT NULL DEFAULT (GETDATE()),
```

Comment:
Changet GETDATE() to SYSDATETIME(). GETDATE() only returns datetime value up to 1/100th of a millisecond accuracy. To get datetime2 value of the current time, use SYSDATETIME() and to get datetimeoffset use SYSDATETIMEOFFSET().